scripts/translate_utils.py

```diff
@@ -3,9 +3,89 @@
 import shutil
 import yaml
 
+class TranslatableAsset:
+    translatable_attributes = []
+
+    def __init__(self, asset: dict):
+        self.asset = asset
+        for key in ASSET_FOLDER_MAPPING:
+            if key in asset:
+                self.asset_type = ASSET_FOLDER_MAPPING[key]
+                break
+
+    def extract_text(self):
```

Comment:
Looks like this was left in by mistake?

Reply:
yes, I'm trying to make sure all strings are translated as it was before

Reply:
Working as before, 236 strings found

Reply:
Yes, but why is it defined twice? Here and on line 21?